os/test/src/PathTests.scala

```diff
@@ -30,6 +30,9 @@ object PathTests extends TestSuite{
           assert(root/"omg" == Path("/omg"))
           assert(rel/"omg" == RelPath("omg"))
           assert(sub/"omg" == SubPath("omg"))
+        } else {
+          //Windows path consistency
+          assert(Path((root/"omg").toIO.toURL.getFile) == root/"omg")
```

Comment:
Shouldn't this test run on all platforms, not just Windows?

Reply:
Originally it was a Windows-specific bug fix but it should also suceed on unix system so I can move it after school